test/scripts/gen_realdata_vectors.py

```
@@ -444,98 +444,98 @@ def generate_cpp(history: list, noto: list, data_txt: list,
+            w('    };')
             w('')
 
         elif dc == 2:  # Hypocenter
             w(f'    // dc={dc} {dc_name_label}: {len(entries)} entries')
-            w(f'    struct HypoExpected {{ uint16_t depth; uint8_t magnitude; uint16_t epicenter; }};')
-            w(f'    static const HypoExpected hypo_expected[] = {{')
+            w('    struct HypoExpected { uint16_t depth; uint8_t magnitude; uint16_t epicenter; };')
+            w('    static const HypoExpected hypo_expected[] = {')
             for _, f in entries:
                 w(f'        {{{f["depth"]}, {f["magnitude"]}, {f["epicenter"]}}},')
             w(f'    }};')
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)